### ISSUE TITLE
bug: Uncaught Error: The sortable option requires table headings. vanilla-dataTables.js:569

### ISSUE THREAD

**Issue by mckaygerhard** [Mobius1/Vanilla-DataTables#4]
**Its there posible to make a table with searchable without sortable!?** it **must** but i can able, 
i set false the sortable option in the data option var but the js forces it to need it

`Uncaught Error: The sortable option requires table headings. vanilla-dataTables.js:569`

the code alfter table was:

```
</table><script src="http://127.0.0.1/~general/codeigniter/appweb/scripts/vanilla-dataTables.js"></script>
<script>
var optionstable306 = { fixedHeight: true, perPage: 50 };
var datatable306 = new DataTable(document.getElementById('table306'), optionstable306);</script>
```

i think based on the code, that searchable need sortable, so testing got right, must set in options that explicit:
` sortable: false, searchable: false, `

so then if there's no posible to set search table without headers, please then autodisable event lauch exception!

so this issue its for:
1) if no header are set, and sortable or/and searchable are true, got exception with message
2) if no options was specified, and no header was set, must disable both, due search implicts sort
3) UPGRADE DOCUMENTATION ABOUT IT

**Comment by Mobius1** [comment]
Thanks for the feedback. A fix is in progress.

**Comment by mckaygerhard** [comment]
all the feedback i'll post in issue #5 this i made only for tecnically organizationall purposes

**Comment by mckaygerhard** [comment]
AWESOME; i'm able to search without headers, so with headers too!

closes #5 and #4 with commit 313ec92 THANKS!
